twbt.cpp

```diff
@@ -144,7 +145,7 @@ long white_texpos, transparent_texpos;
 
 long cursor_small_texpos;
 
-static bool enabled;
+DFHACK_PLUGIN_IS_ENABLED(enabled);
```

Comment:
No, it just means that `plug` will report whether TWBT is enabled (it won't actually implement `enable`/`disable` commands).